examples/C++/DDS/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set(fastrtps_FOUND TRUE)
```

Comment:
Examples can be compiled individually by the user, and in that case, if Fast-RTPS wasn't installed in the system, the user is responsible to indicate CMake where it can be found.

Reply:
:+1: